### ISSUE TITLE
docker base image problem?

### ISSUE THREAD

**Issue by sysdl132** [univention/ucs-appliance-container#5]
https://github.com/sysdl132/ucs-appliance-container/commit/e9c36e4893be0bb68a5e35dc755e1b1a5c26da57

I think you need to use `` univention/univention-corporate-server-debootstrap`` instead of ``univention-corporate-server-debootstrap`` , or the build may fail? 

![image](https://user-images.githubusercontent.com/54708046/124443748-86c56c80-ddb0-11eb-830e-7ead00b97cde.png)

**Comment by JOduMonT** [comment]
Yes almost every example they have forget the univention/ which is required since they are not official image. I could find the same error here: https://hub.docker.com/r/univention/univention-corporate-server

**Comment by sysdl132** [comment]
Automatic closed - inactivity
